localgov_events.install

```diff
@@ -73,3 +73,34 @@ function localgov_events_update_8002() {
     $config->save(TRUE);
   }
 }
+
+/**
+ * Expanding Events permissions.
+ *
+ * Grant taxonomy term permissions to Editor role.
+ */
+function localgov_events_update_10001($is_syncing) {
```

Comment:
MT: Good to do this as the default for new installs; but not updating existing sites.

Reply:
OK, removed.